src/cljs/commiteth/core.cljs

```diff
@@ -33,13 +34,46 @@
             [:i.close.icon {:on-click #(rf/dispatch [:clear-flash-message])}]
             [:p message]]])))))
 
+(defn warning-icon []
+  (let [eth-address (rf/subscribe [:get-in [:user :address]])
+        repo-count (rf/subscribe [:get-in [:user :repo_count]])  
+        hide-address-warning (rf/subscribe [:get-in [:user :hide_address_warning]])
+        hide-gh-repos-warning (rf/subscribe [:get-in [:user :hide_gh_repos_warning]])
+        warning-open? (rf/subscribe [:warning-open?])]
```

Comment:
These `:get-in` subscriptions can lead to problems should we ever change the structure of our app-db. Not a big deal for now but worth keeping in mind. Suggestions: Have a `:user` subscription that returns full user map.